tests/test_skill.py

```diff
@@ -16,37 +18,37 @@ def hello_skill(self, message):
         message.respond("Hello")
 
 
-class TestSkill(unittest.TestCase):
-    """Test the opsdroid skill base class."""
+configure_lang({})
+
+
+def test_init():
+    config = {"example_item": "test"}
+    skill = Skill(None, config)
+    assert skill.opsdroid is None
+    assert skill.config["example_item"] == "test"
+
 
-    def setUp(self):
-        configure_lang({})
+def test_init_erroneous_property():
+    """Test that properties that raise an exception don’t mess up initialisation"""
```

Comment:
Docstrings need to finish with a dot
```suggestion
    """Test that properties that raise an exception don’t mess up initialisation."""
```